UnityProject/Assets/Scripts/NPC/MobSprite.cs

```diff
@@ -172,12 +173,50 @@ public void SetBurningOverlay()
 			burningOverlay.StartOverlay(directional.CurrentDirection);
 		}
 
+		/// <summary>
+		/// Enable the electrocuted overlay for the player's mob.
+		/// </summary>
+		/// <param name="time">If provided and greater than zero, how long until the electrocuted overlay is disabled.</param>
+		public void EnableElectrocutedOverlay(float time = -1)
+		{
+			if (electrocutedOverlay == null)
+			{
+				AddOverlayGameObjects();
+			}
+
+			if (time > 0)
+			{
+				StopElectrocutedOverlayAfter(time);
```

Comment:
```suggestion
				StartCoroutine(StopElectrocutedOverlayAfter(time));
```

Reply:
Ah, so that's how to suggest changes. neat!

Reply:
Cheers!